src/components/chat/userMessage.tsx

```diff
@@ -1,22 +1,34 @@
+'use client';
+import WonkyError from '@/lib/error/wonkyError';
+import WonkyErrorBoundary from '@/lib/error/wonkyErrorBoundary';
+
 import { UserPortrait } from './userPortrait';
 
 export const UserMessage = ({
   user,
-  children,
+  content,
 }: {
   user?: string;
-  children: React.ReactNode;
+  content: string;
 }) => {
   return (
     <div className='row mb-3'>
       <div className='col-2 col-sm-1 mb-2'>
-        <UserPortrait />
+        <WonkyErrorBoundary>
+          <UserPortrait />
+        </WonkyErrorBoundary>
       </div>
       <div className='col-10'>
         <p className='chat-name'>
           <strong>{!!user ? user : `You: `}</strong>
         </p>
-        <p>{children}</p>
+        <WonkyErrorBoundary
+          fallback={
+            <WonkyError type='alert' thereWasAnErrorLoadingThe='question' />
+          }
+        >
+          <p>{content}</p>
```

Comment:
just curious why change from children to prop?  not something the error boundary made you do right? just a stylistic decision?